index.js

```diff
@@ -139,43 +139,45 @@ btnClear.addEventListener('click', function () {
 
 btnSubtraction.addEventListener('click', function () {
     operation = '-';
-    calcScreen.innerHTML = '-';
+    calcScreen.innerHTML = operation;
     console.log(operation);
 });
 
 btnAddition.addEventListener('click', function () {
     operation = '+';
-    calcScreen.innerHTML = '+';
+    calcScreen.innerHTML = operation;
     console.log(operation);
 })
 
 btnDivision.addEventListener('click', function () {
     operation = "/";
-    calcScreen.innerHTML = '÷';
+    calcScreen.innerHTML = operation;
     console.log(operation);
 })
 
 btnMultiplication.addEventListener('click', function () {
     operation = '*';
-    calcScreen.innerHTML = '×';
+    calcScreen.innerHTML = operation;
     console.log(operation);
 })
 
 btnEqual.addEventListener('click', function () {
     let result;
-    if (operation === "+") {
-        result = nr1 + nr2;
-    }
-    if (operation === '-') {
-        result = nr1 - nr2;
-    }
-    if (operation === "*") {
-        result = nr1 * nr2;
-    }
-    if (operation === "/") {
-        result = nr1 / nr2;
+    switch (operation) {
+        case "+":
+            result = nr1 + nr2;
+            break;
+        case "-":
+            result = nr1 - nr2;
+            break;
+        case "*":
+            result = nr1 * nr2;
+            break;
+        case "/":
+            result = nr1 / nr2;
+            break;
```

Comment:
Missing `default` case. It's usually a good pattern to always have the `default` branch.

Here, in case of the `default` branch you can use the `alert` method to signal to the user that something wrong happened, and they should try refreshing the page or getting in touch with you.